src/app/views/welcome/welcome.vue

```diff
@@ -0,0 +1,49 @@
+import { darken } from 'polished';
+<template>
+	<div class="-welcome-page">
+		<div class="-welcome-col">
+			<section class="-message">
+				<p class="lead">
+					<translate>
+						Before you go game spelunking, let's customize your profile.
```

Comment:
Hmm, let's just keep it super simple right now (I can improve later with more fun copy): `Let's get you set up!`